components/blocks/FeatureGrid.tsx

```diff
@@ -37,27 +37,23 @@ const Feature = ({ data, index, id }) => {
         {actions && <Actions items={actions} />}
       </div>
     </a>
-
-  ) : (
-    <Link
-      href={formattedUrl}
-      className="h-full w-full group block py-6 px-8 md:py-9 md:px-11 lg:py-12 lg:px-14 rounded-sm bg-gradient-to-br from-white via-white to-white/50 shadow-[inset_0_0_0_1px_rgba(223,219,252,0.15),_0_0_1px_1px_rgba(223,219,252,0.5)] transition duration-500 hover:scale-105 hover:bg-gradient-to-br hover:from-orange-200 hover:via-orange-400 hover:to-orange-600 hover:z-20"
-      style={{ textDecoration: 'none', overflow: 'visible' }}
-      id={id}
-    >
-      <div
-        data-tina-field={tinaField(data, 'headline')}
-        className="flex flex-col gap-4"
-      >
-        {headline && (
-          <h3 className="text-2xl md:text-xl lg:text-2xl font-tuner leading-tight text-transparent bg-gradient-to-br from-blue-700/80 via-blue-900/90 to-blue-1000 bg-clip-text mb-2 group-hover:text-white break-words">
-            {headline}
-          </h3>
-        )}
-        {text && <p className="group-hover:text-white">{text}</p>}
-        {actions && <Actions items={actions} />}
-      </div>
-
+  ) : 
+  (
+    <Link 
+    href={formattedUrl} 
+    className="h-full w-full group block py-6 px-8 md:py-9 md:px-11 lg:py-12 lg:px-14 rounded-sm bg-gradient-to-br from-white via-white to-white/50 shadow-[inset_0_0_0_1px_rgba(223,219,252,0.15),_0_0_1px_1px_rgba(223,219,252,0.5)] transition duration-500 hover:scale-105 hover:bg-gradient-to-br hover:from-orange-200 hover:via-orange-400 hover:to-orange-600 hover:z-20"
```

Comment:
Lots of tailwind classes - Can we move repeated stuff to tailwind config?

Reply:
This will be addressed in https://github.com/tinacms/tina.io/issues/1823